app/controllers/session_controller.rb

```diff
@@ -0,0 +1,16 @@
+class SessionController < ApplicationController
+  # skip_before_action :verify_authenticity_token, only: [:create]
+
+  #GET /session/new
+  def new
+  end
+
+  #POST /session
+  def create
+    user = User.find_by(email: params[:email])
+    if user && user.authenticate(params[:password])
+      session[:user_id] = user.id
```

Comment:
do we need a session at this point?

Reply:
Yes. we need that because we need the id to link to the post page